protos/sigstore_trustroot.proto

```diff
@@ -0,0 +1,86 @@
+// Copyright 2022 The Sigstore Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+package dev.sigstore.trustroot.v1;
+
+import "sigstore_common.proto";
+
+option go_package = "github.com/sigstore/proto/trustroot/v1";
+option java_package = "dev.sigstore.proto.trustroot.v1";
+option java_multiple_files = true;
+option java_outer_classname = "TrustRootProto";
+
+// TransparencyLogInstance describes the immutable parameters from a
+// transparency log.
+// See https://www.rfc-editor.org/rfc/rfc9162.html#name-log-parameters
+// for more details.
+// The incluced parameters are the minimal set required to identify a log,
+// and verify an inclusion promise.
+message TransparencyLogInstance {
+        // The base URL at which can be used to URLs for the client.
+        string base_url = 1;
+        // The hash algorithm used for the Merkle Tree.
+        dev.sigstore.common.v1.HashAlgorithm hash_algorithm = 2;
+        // The public key used to verify signatures generated by the log.
+        // This attribute contains the signature algorithm used by the log.
+        dev.sigstore.common.v1.PublicKey public_key = 3;
```

Comment:
Do we expect more than public_key being embedded in the TUF target? Today we only have the public_key in the TUF root, so just wondering if we're going to bake some of this information into our TUF targets? For example, we could serialize this into the TUF target for a given Rekor instance, so just wondering out loud. Jotted it down here, but applicable to CA below as well.

Reply:
I can't say for sure, but I _think_ so. As an example the `base_url` corresponds to the metadata field `uri` in the TUF target. We are talking about adding a time range for the log too, which would then populate the same fields in the linked `public_key`. The same reasoning goes for CA too of course. 
This is not the complete set of attributes, some are missing still like the hash and signature algorithm for the log, or the distinguished name for a CA. As they are relevant from a verification perspective, we should think how such values are communicated. The distinguished may be part of the policy decision though.